### ISSUE TITLE
Auto Payments working from Stripe.

### ISSUE THREAD

**Issue by mohitsharma-22** [glific/glific#1981]
After 1st October 2021, the Auto payments stopped working from Stripe due to RBI rule.

Can we please check if there is any fix, we have so that the auto payments can be resumed?

Sharing the updated link for communication from Stripe.
https://support.stripe.com/questions/india-network-tokenization-faqs-and-upcoming-stripe-solutions

**Comment by mohitsharma-22** [comment]
@pankaj-ag , Yes, we still are not getting Payment automatically from credit cards. Please see the tokenization mentioned in the document above and see if we can implement it to get the payments automatically.

**Comment by dlobo** [comment]
let figure out how to Handke recurring in Q3
